native/cocos/math/Vec3.cpp

```diff
@@ -27,6 +27,36 @@
 #include "math/MathUtil.h"
 #include "math/Quaternion.h"
 
+#if (CC_PLATFORM == CC_PLATFORM_ANDROID)
+    #include <cpu-features.h>
+#endif
+
+#if (CC_PLATFORM == CC_PLATFORM_IOS)
+    #if defined(__arm64__)
+        #define USE_NEON64
+        #define INCLUDE_NEON64
+    #elif defined(__ARM_NEON__)
+        #define USE_NEON32
+        #define INCLUDE_NEON32
+    #else
+    #endif
+#elif (CC_PLATFORM == CC_PLATFORM_ANDROID)
+    #if defined(__arm64__) || defined(__aarch64__)
+        #define USE_NEON64
+        #define INCLUDE_NEON64
+    #elif defined(__ARM_NEON__)
+        #define INCLUDE_NEON32
+    #endif
+#endif
+
+#if defined(USE_NEON64) || defined(USE_NEON32) || defined(INCLUDE_NEON32)
+    #include "enoki/array.h"
+    #ifndef ENOKI_ARM_NEON
```

Comment:
Where defines this macro?

Reply:
ENOKI_ARM_NEON is defined in `enoki/fwd.h`, I checked it here for ensuring neon is enabled in enoki.